.github/workflows/image-push-main.yml

```diff
@@ -0,0 +1,52 @@
+name: "Image push main"
+on:
+  push:
+    branches:
+      - main
+jobs:
+  image-build-push:
+    name: Image build and push
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set repository as lower-case output variable
```

Comment:
isn't the repo name already lowercase?

Reply:
Yes but the organization name (Mellanox) is not

Reply:
ahh, that's right!